ksqldb-engine/src/test/java/io/confluent/ksql/engine/KsqlEngineTest.java

```diff
@@ -137,7 +137,7 @@ public void setUp() {
 
   @After
   public void closeEngine() {
-    ksqlEngine.close();
+    ksqlEngine.close(true);
```

Comment:
It looks like it's still failing. Is it possible this still isn't ultimately calling `close` on the `KafkaStreams`? (just fyi, it's `KafkaStreams#close` we need to call, since that unlocks the dir, not necessarily `KafkaStreams#cleanUp`. But it's probably good to also call `#cleanup` in tests as well, since it just cleans up whatever state is left behind.